editor/header/tools/preview-button.js

```diff
@@ -16,10 +15,14 @@ import { getEditedPostPreviewLink } from '../../selectors';
 
 function PreviewButton( { link } ) {
 	return (
-		<Button href={ link } target="_blank">
-			<Dashicon icon="visibility" />
+		<IconButton
+			href={ !! link ? link : undefined }
```

Comment:
When I disable the button, the link is equal to `""` which triggers a `a` and the pseudo-class `disabled` don't work on links.